server/services/findAndReferService.test.ts

```diff
@@ -0,0 +1,18 @@
+import FindAndReferService from './findAndReferService'
+import config from '../config'
+import RestClient from '../data/restClient'
+import MockRestClient from '../testutils/mockRestClient'
+
+// wraps mocking API around the class exported by the module
+jest.mock('../data/restClient')
+
+describe('Find and Refer Service', () => {
+  const findAndReferService = new FindAndReferService(config.apis.findAndReferService)
+  const restClientMock = new MockRestClient(config.apis.findAndReferService) as jest.Mocked<RestClient>
```

Comment:
is there any reason we pass in the config here rather than just referencing it in the Service class?

Reply:
I was following a pattern in the old project, but happy to switch it